.github/workflows/ci.yml

```
@@ -83,10 +83,12 @@ jobs:
           bundle exec parallel_rspec -n $PARALLEL_TESTS_CONCURRENCY -e './bin/parallel_tests'
       - name: Check for missing annotations
         run: bundle exec annotate
-      - name: Check for untracked changes in app and spec directories
+      - name: Check for missing doc changes
+        run: ./bin/update-docs && git diff
```

```suggestion
        run: ./bin/update-docs
```

is it necessary to git diff?

---

It was for debugging, but I left it since when everything is ok nothing shows, but when we get an error it specifies which changes it has, making it easier to fix.